Added clear command

mrmike opened the pull request:
Added clear comment for clearing app data like cache, databases and shared preferences.
To use clear command rooted device or emulator is required.

Commit:
Added clear command

Summary: Added clear command for clearing app data
like databases, shared preferences and cache.

Commit:
Fixed syntax typo

mttkay commented:
Love the idea. One thing I might change: it looks like you're relying on default_package to be set, but it might be nil. We should probably catch that case and expect a package argument instead?

mrmike commented:
Sure, I will change that.

Commit:
Handling clear arguments

Summary: Handling arguments of clear command.
Acceptable is one argument or zero when default package is set.

mrmike commented:
Hi Matthias,

I've added argument handling for clear command.

mttkay commented:
Thanks!
